core/arch/x86/decode_table.c

```diff
@@ -1483,6 +1483,10 @@ const instr_info_t * const op_instr[] =
 #define KEw TYPE_K_EVEX, OPSZ_2
 #define KEd TYPE_K_EVEX, OPSZ_4
 #define KEq TYPE_K_EVEX, OPSZ_8
+#define VEvs TYPE_V, OPSZ_16_32_evex64
```

Comment:
Why is there a capital E on these types?  The type is `TYPE_V` so they should just be `Vvs`, `Vvd`, etc.

Reply:
Offline discussion: Changed to Ves, Ved, Wes, Wed.